manifests/init.pp

```diff
@@ -46,6 +46,8 @@
 # @param ensure_absent_flags Array of flags to use with authconfig when service
 #   is disabled.
 #
+# @param authselect_profile  Name of authselect profile to use
```

Comment:
What is this for? Is it something new in RHEL8? Could you link to some of the RHEL8 documentation that explains it.

Reply:
No, this is in Fedora also.
Authselect allows you to change between authentication "profiles"
Fedora/RHEL8 ships with a handful of predefined profiles including an sssd profile.
But you can also define your own.
We use a locally defined profile with does sssd but adds MFA as well.